src/ui/services/networkService.ts

```diff
@@ -3,9 +3,13 @@
 
 import { EventEmitter } from "events";
 import StaticService from "./staticService.ts";
+import f from "../../../ConfigReader.ts";
 
 const io = <SocketIOClientStatic>require("socket.io-client");
-
+var player_id;
+var pathToLolExe;
+var pathToLolFolder;
+var child_process = require('child_process');
```

Comment:
Actually scratch that, it's alright to match names with whatever you're importing